src/diffusers/schedulers/scheduling_flow_match_euler_discrete.py

```diff
@@ -129,7 +129,31 @@ def scale_noise(
         if self.step_index is None:
```

Comment:
I think we can also remove this here 

Let's remove it and run the example for img2img to see if the results match from `main` and this branch